### ISSUE TITLE
Unable reproduce the performance of SUN-RGBD

### ISSUE THREAD

**Issue by encore-zhou** [zaiweizhang/H3DNet#5]
Hi, I try to training the H3DNet on SUN-RGBD dataset(using the default configs), but my reproduced performance is lower than the paper reported. Are there others tricks that I missed?

**Comment by zaiweizhang** [comment]
Hi, I am able to reproduce the results on my machines. However, I do observe some unstable behaviors. With some new experiments, I think the following parameters may give you a more stable performance.

Use batch size 16, and use the following learning rate setting:
BASE_LEARNING_RATE = 0.01 or 0.001
FLAGS_LR_DECAY_STEPS = '100, 160, 220, 260, 360'
FLAGS_LR_DECAY_RATES = '0.1, 0.1, 0.1, 0.1, 1'

I am able to achieve around 60 mAP@0.25 and 39 mAP@0.5.

What are the results you are getting?

**Comment by encore-zhou** [comment]
Thank you so much. Let me try again with the new configs.

**Comment by zaiweizhang** [comment]
Close it for now. Feel free to reopen it.
